ext/lib/crypto/mbedtls/zephyr_init.c

```diff
@@ -20,6 +21,8 @@
 #endif /* CONFIG_MBEDTLS_CFG_FILE */
 #endif
 
+K_APPMEM_PARTITION_DEFINE(k_mbedtls_partition);
```

Comment:
bug: this needs to live in kernel memory, but since we are pulling in the .a file, this partition definition is instead living in app shared memory. need to fix this.